src/pt/atemporal/src/eu/kanade/tachiyomi/extension/pt/atemporal/Atemporal.kt

```diff
@@ -0,0 +1,53 @@
+package eu.kanade.tachiyomi.extension.pt.atemporal
+
+import android.webkit.CookieManager
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+import okhttp3.Cookie
+import okhttp3.CookieJar
+import okhttp3.HttpUrl
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class Atemporal : Madara(
+    "Atemporal",
+    "https://atemporal.cloud",
+    "pt-BR",
+    dateFormat = SimpleDateFormat("d 'de' MMMM 'de' yyyy", Locale("pt", "BR")),
+) {
+    override val client = network.cloudflareClient.newBuilder()
+        // Force visited=true cookie so we can use search
```

Comment:
use `cookieinterceptor` lib to set cookie instead

Reply:
I tried using the cookie interceptor, but that requires you to open the WebView first. Is that the intended behavior?
```kt
// Cookie required for search
private val cookieInterceptor = CookieInterceptor(baseUrl.toHttpUrl().host, "visited" to "true")

override val client = network.cloudflareClient.newBuilder()
    .addNetworkInterceptor(cookieInterceptor)
    .build()
```

Reply:
no? it should work without opening webview 🤔
try adding as regular interceptor instead of network interceptor

Reply:
My bad, ran into the issue described in [the contributing guidelines](https://github.com/keiyoushi/extensions-source/blob/main/CONTRIBUTING.md#running):
> [...] you might face issues such as Android Studio running an older version of the extension [...]